packages/app-form-builder/src/admin/plugins/formDetails/previewContent/HeaderComponents/DeleteForm.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { useApolloClient } from "@apollo/react-hooks";
+import { useRouter } from "@webiny/react-router";
+import { useSnackbar } from "@webiny/app-admin/hooks/useSnackbar";
+import { IconButton } from "@webiny/ui/Button";
+import { Tooltip } from "@webiny/ui/Tooltip";
+import { ConfirmationDialog } from "@webiny/ui/ConfirmationDialog";
+import { ReactComponent as DeleteIcon } from "../../../../icons/delete.svg";
+import * as queries from "../../../../graphql";
+import { removeFormFromListCache } from "~/admin/views/cache";
+import { usePermission } from "~/hooks/usePermission";
+import { FbRevisionModel } from "~/types";
+
+interface DeleteRevisionProps {
+    form: FbRevisionModel;
+    revision: FbRevisionModel;
+}
+const DeleteRevision: React.FC<DeleteRevisionProps> = ({ form, revision }) => {
```

Comment:
I saw your comment:

>  Also updated Delete feature for Form Preview tab, now it acctually deletes entire form instead of selected revision.

What do you mean by this?

I mean, users should be able to delete just a specific revision of the form. Not sure I understand what's going on here. 🤔